test/steps/results.js

```diff
@@ -48,27 +49,31 @@ async function seeTheResultsIn(
     label: string,
     table: Array<Object>,
 ): Promise<void> {
-    const getText = element => element.getText();
     const keyToClass = (key) => key.match(/[(](.*)[)]/)[1];
     const selector = label && `.${label.replace(/ /g, "")}`;
+    console.log(table);
```

Comment:
Is this still needed?

Reply:
oops omg I keep forgetting to remove my console.logs

Reply:
Nah you're all good, happens to the best of us, I forget them all the time